composer.json

```diff
@@ -12,8 +12,15 @@
     "composer/installers": "1.7.0",
     "maxmind-db/reader": "1.6.0",
     "woocommerce/woocommerce-blocks": "2.5.10",
-    "woocommerce/woocommerce-rest-api": "1.0.6"
+    "woocommerce/woocommerce-rest-api": "1.0.6",
+    "woocommerce/woocommerce-admin": "dev-try/downgrade-autoloader as 0.24.0"
```

Comment:
Is this the correct version we plan to include?

Reply:
We need to update the autoloader version in `master` first.

Reply:
@jeffstieler what version do we need to update it to? Background here p90Yrv-1wu-p2

Reply:
@timmyc we don't yet have a PR to update the version in WC-Admin, but it needs to change to `^1.2.0` to match core WC and the blocks repo.

Reply:
So just a quick check, it looks like `master` [is set to `^1.2.0`](https://github.com/woocommerce/woocommerce/blob/master/composer.json#L11) so thinking this requirement has been satisfied.

Reply:
It needs to be updated to match in the WC-Admin repo as well.

Reply:
https://github.com/woocommerce/woocommerce-admin/pull/3667/

Reply:
Updated in 88ec069.

Reply:
v0.25.0 has been published, and jetpack-auotloader in wc-admin is the same version as here in Woo core. ( cc @peterfabian  )

Reply:
Thanks for the update! The current inclusion uses the source package that is getting cloned from GitHub, not the dist package. Can we switch to dist/zip package, similarly to how Blocks is doing it? Otherwise wc-admin is 47MB when cloned to `packages/`.

Reply:
Correct I believe we were pointed at GitHub to resolve the issue we were having with the mis-match of the jetpack autoloader. So we should be able to point at packagist ( as @claudiosanches mentions below )

Reply:
Solved!